content/admin/org_settings/integrations/GenericWebhook.md

```diff
@@ -16,40 +16,46 @@ Contrast supports generic webhook integration, which allows you to receive notif
 * In the row for Generic Webhook, click the button to **Connect**.
 * Name the webhook, and paste the URL in the designated field.
 * Select the application(s) that you want to filter.
-* If you want to complete the Payload field, enter the `title` and `message`. Sample code:
+* Enter a payload such as:
 
 ```javascript
 {
-	'title':   'Contrast Security Notification',
-	'message': 'Test User commented on a Insecure JSP Placement vulnerability in WebGoat. \"Fixed in CVE-2015\"'
+	'title': $Title,
+	'message': $Message
 }
 ```
 You can also add placeholders in the payload so that Contrast will fill in for different notifications - a new application, server, vulnerability, etc. 
 
-The placeholders available for use are:
-
-* Title
-* Message
-* Application Name
-* Vulnerability Tags
-* Application ID
-* Server Name
-* Server ID
-* Organization ID
-* Severity
-* Status
-* Environment
-* Vulnerability Rule
-* Vulnerability ID
-* Event Type
-
-If you were using the Generic Webhook to create an integration, the sample payload could be as follows:
+The placeholders available for use that will always return a value:
+
+* `$Title` - The title of the vulnerability
+* `$Message` - A human-readable summary of the event.
+* `$EventType` - The type of event which will be one of NEW_VULNERABILITY, NEW_ATTACK, ATTACK_UPDATE, NEW_ATTACK_APPLICATION, ATTACK_END, VULNERABILITY_CHANGESTATUS_OPEN, VULNERABILITY_CHANGESTATUS_CLOSED:, SERVER_OFFLINE, NEW_ASSET, EXPIRING_LICENSES, NEW_VULNERABILITY_COMMENT, ATTACK_EVENT_COMMENT.
```

Comment:
Is the ":" required that's currently attached to "VULNERABILITY_CHANGESTATUS_CLOSED"?

Reply:
Nope. Good catch!